README.md

```diff
@@ -64,51 +64,43 @@ provider "env0" {
 
 ## Dev setup
 
-To build locally, you can use the `./build.sh` script.
-The rest of the steps below are relevant if you would like to use the provider outside of the test harness (the test harness performs the steps described here).
-
-To use local binary version, you'll need to create a local terraform provider repository.
-The simplest way to do so would be to create a folder on your disk.
-Under that folder, copy the built provider binary to `terraform-registry.env0.com/env0/env0/6.6.6/linux_amd64/terraform-provider-env0` (note to replace linux_amd64 if using a different platform).
-Then, create a `terraform.rc` file (location doesn't matter).
-The content of said file should look like so:
-
+### Build 
+- Use the `./build.sh` script.
+- The output binary is called `terraform-provider-env0`
+
+### Run local version of the provider
+- Build - `./build.sh`
+- Create the plugins folder - `mkdir -p ~/.terraform.d/plugins/terraform.env0.com/local/env0/6.6.6/darwin_amd64`
+- Copy the built binary - `cp ~/env0/terraform-provider-env0/terraform-provider-env0 ~/.terraform.d/plugins/terraform.env0.com/local/env0/6.6.6/darwin_amd64` (Replace `darwin` with `linux` on Linux)
+- Require the local provider in your `main.tf` - 
 ```
-provider_installation {
-  filesystem_mirror {
-    path    = "<absolute path to local repository folder>"
-    include = ["terraform-registry.env0.com/*/*"]
+terraform {
+  required_providers {
+    env0 = {
+      version = "6.6.6"
```

Comment:
```suggestion
      version = "x.x.x"
```

Reply:
It has to be a valid semver and also needs to be like the folder you copy the bin to